ansible/0001-ansible-Adding-support-for-droplet-vlans.patch

```diff
@@ -0,0 +1,169 @@
+From c3ecc39c784ef90efbbc524832ef84a59ef7d9d8 Mon Sep 17 00:00:00 2001
```

Comment:
What is this patch file? Did it get here by accident?

Reply:
removed. thanks